README.md

```diff
@@ -265,6 +265,14 @@ Specify the following `oauth.*` properties:
 Some authorization servers don't provide the `iss` claim. In that case you would not set `oauth.valid.issuer.uri`, and you would explicitly turn off issuer checking by setting the following option to `false`:
 - `oauth.check.issuer` (e.g. "false")
 
+You can enforce audience check, which is an OAuth2 mechanism to limit access to tokens that are explicitly issued for use by your protected server.
```

Comment:
I think "limit access" is a bit confusing. IIUC it's not that there is something which can't _access_ the tokens, it's that the tokens are rejected as invalid because they weren't issued to be used for that Kafka broker (acting in the role of resource server). Maybe "...which is an OAuth2 mechanism to prevent validation of tokens that weren't explicitly issued for use by your protected server." would be better phrasing.

Also, normal English usage would be "You can enforce audience checking".

Reply:
OTOH when I hear 'enforce audience checking' I specifically think of this functionality being enabled vs. not being enabled. I don't think about the process of validating the token being triggered and failing. Rather I think about the process of validating not being attempted at all.

Reply:
Ah, sorry I misunderstood your last sentence. So with regards to 'limit access', what about saying it like this:

"...which is an OAuth2 mechanism to prevent *authentication* of tokens that weren't explicitly issued for use by your protected server."

Reply:
Or maybe it should be:

"...which is an OAuth2 mechanism to prevent *successful authentication with* tokens that weren't explicitly issued for use by your protected server."

Reply:
SGTM.

Reply:
Slightly reworded to fit with the line that follows:

"You can enforce audience checking, which is an OAuth2 mechanism to prevent *successful authentication with* tokens **unless they are** explicitly issued for use by your resource server."